pkg/operator/ceph/cluster/mgr/dashboard.go

```diff
@@ -51,7 +51,8 @@ const (
 )
 
 var (
-	dashboardInitWaitTime = 5 * time.Second
+	dashboardInitWaitTime        = 5 * time.Second
+	removeMgrDaemonConfiguration = true
```

Comment:
do we need this globally?

Reply:
I think the answer is yes because we need this to be stored between reconcile calls.